src/EnergyPlus/PluginManager.cc

```diff
@@ -201,7 +201,7 @@ void PluginManager::setupOutputVariables([[maybe_unused]] EnergyPlusData &state)
                                         sUpdateFreq,
                                         sAvgOrSum,
                                         thisObjectName,
-                                        {},
+                                        "",
```

Comment:
Is this a better option?  All the changes from strings to enums look great, this just seems like an out of place change.  Could you clarify?